styles/globals.css

```diff
@@ -365,6 +365,72 @@ blockquote {
   color: #4b5563;
 }
 
+/* Footer social hover effects */
+
+#twitter:hover,
```

Comment:
given the use of tailwind css can we move the styles from the global css file to inline tailwind styles?

Reply:
This I am unsure how to do based on the alternating gradients. Well besides verbose if statements 🤔

Reply:
https://tailwindcss.com/docs/hover-focus-and-other-states

https://tailwindcss.com/docs/reusing-styles

These might help. 

i am fine with using the css though. I would probably do the same. 

While using ids in global css will work, do you think it will be better for maintainability to move to a css module?

Something like Footer.module.css
// styles in here convert to classes ? since locally scoped we don't need an id now. class name will suffice. 

in Footer component import:
` import styles from './Footer.module.css';`

className={`p-1 rounded-md ${styles[item.name.toLowerCase()]}`}

something like this. I leave to you to choose implementation - your global styles will also work . 

https://nextjs.org/docs/app/building-your-application/styling/css-modules

Reply:
I just made a pr, I used the @apply method, hope its an acceptable alternative.